frontend/src/components/camp/Picasso.vue

```diff
@@ -192,13 +192,16 @@ export default {
         scheduleEntry.activity().title
     },
     getActivityColor (scheduleEntry, _) {
-      if (this.isActivityLoading(scheduleEntry)) return 'grey lighten-2'
+      if (this.isCategoryLoading(scheduleEntry)) return 'grey lighten-1'
       const color = scheduleEntry.activity().category().color
       return isCssColor(color) ? color : color + ' elevation-4 v-event--temporary'
     },
     isActivityLoading (scheduleEntry) {
       return this.activitiesLoading || (scheduleEntry.activity()._meta ? scheduleEntry.activity()._meta.loading : false)
     },
+    isCategoryLoading (scheduleEntry) {
+      return (scheduleEntry.activity().category()._meta ? scheduleEntry.activity().category()._meta.loading : false)
```

Comment:
Minor: ginge auch einfacher:
```suggestion
      return scheduleEntry.activity().category()._meta?.loading ?? false
```